src/pages/[language]/[slug]/index.query.graphql

```diff
@@ -21,6 +21,7 @@ query LandingPage($locale: SiteLocale, $slug: String) {
         image {
           ...image
         }
+        imagePosition
```

Comment:
I didn't like the word `inverse` like we use in the old dato block, but I can also use that if you prefer

Reply:
Fully agree, in most situations I prefer to use 'options' over a single boolean because it allows to add more variants in the future.